src/App/Main.fs

```diff
@@ -524,6 +524,8 @@ let private subscriptions (model: Model) =
 
     Cmd.ofSub sub
 
+Browser.navigator.serviceWorker.register("/service-worker.js") |> ignore
```

Comment:
Do we need to do this in the worker code (it's the worker which does most of the downloads) as well, or is it enough to just put it here?

Reply:
Just here is enough, the service worker has a wide scope